runtime/jcl/common/criusupport.c

```diff
@@ -283,16 +320,33 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		resultType = vm->criuSupportSuccess;
 
 releaseExclusive:
+		vmFuncs->releaseExclusiveVMAccess(currentThread);
+closeWorkDirFD:
+		if (0 != close(workDirFD)) {
+			systemReturnCode = errno;
+			if (TRUE == isAfterCheckpoint) {
```

Comment:
`if (TRUE == isAfterCheckpoint) { ` can just be `if (isAfterCheckpoint) {`

Reply:
Ok